.azure-pipelines/client.yml

```diff
@@ -87,16 +90,28 @@ jobs:
         inputs:
           command: test
           projects: '$(ProjectFile)'
-          arguments: --filter TestCategory!=Live
+          arguments: --filter TestCategory!=Live --logger trx
+          publishTestResults: false
 
       - task: DotNetCoreCLI@2
         displayName: 'Build & Test (with live tests)'
         condition: ne(variables['System.TeamProject'], 'public')
+        #enabled: false
         env:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
           AZ_CONFIG_CONNECTION: $(AzConfigConnectionString)
         inputs:
           command: test
-          projects: '$(ProjectFile)'
\ No newline at end of file
+          projects: '$(ProjectFile)'
```

Comment:
You need to add the publishTestResults = false here as well as pass the logger argument.